### PR TITLE
Structure

### LINKED ISSUE #4: Add folder structure for appengine

### PR DESCRIPTION
**Pull request by Kaleab470**
Changed folder structure. Closes #4

### CODE REVIEW

File: maps/pom.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  <build>
+    <plugins>
+      <!-- Provides `mvn package appengine:run` for local testing
+           and `mvn package appengine:deploy` for deploying. -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+        <configuration>
+          <deploy.projectId>kaleabg-step-2020</deploy.projectId>
```
Comment (damargulis) [inline maps/pom.xml:47]: Add a TODO to change this to the step33 one, once the billing gets set up.
If you create a github issue for this, you can link the TODO with the issue
```
<!-- TODO(https://githubIssueLink/#4): Update this id -->
```

Comment (Kaleab470) [reply maps/pom.xml:48]: Done!

File: maps/pom.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  <build>
+    <plugins>
+      <!-- Provides `mvn package appengine:run` for local testing
+           and `mvn package appengine:deploy` for deploying. -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+        <configuration>
+        <!-- TODO(https://githubIssueLink/#6): Update this id -->
```
Comment (damargulis) [inline maps/pom.xml:47]: I meant for this to be the actually url to the issue, whatever that is haha, githubIssueLink was just my shorthand

Comment (Kaleab470) [reply maps/pom.xml:47]: haha i actually didn't pay attention to the base url and just changed the issue number :)

**Pull request merged**